### DISCUSSION TITLE
Clarion OS Logo Challenge

### DISCUSSION THREAD

**Discussion by bytemaster** [bytemaster/clarion#14] [Q&A] [answered]
I am looking for creative individuals who would like to contribute a logo to the cause. 

I am willing to *give* 100 EOS to someone who can produce a logo that I find acceptable. This is not a commitment to buy. Your logo submissions will be considered a gift to the cause and must be released to the community under Creative Commons prior to being considered for appreciation. My 100 EOS should be considered a gift of appreciation to the individual(s) that produce something usable.

## History on Clarion OS Name

**clar·​i·​on | \ ˈkler-ē-ən  , ˈkla-rē- \**

Definition of clarion (noun)
- a medieval trumpet with clear shrill tones
- the sound of or as if of a clarion

Definition of clarion (adjective) 
- brilliantly clear
- loud and clear

Clarion is commonly used with the phrase "a clarion call" which means a loud, clear, request for action.

<img width="463" alt="image" src="https://user-images.githubusercontent.com/347290/110019583-a3647180-7cf6-11eb-8c60-3a0936aa1717.png">

## Alternative Themes 

Clarion OS is all about communication and broadcasting so everyone in the world who wants to tune in can hear. This is similar to the idea that Tesla had for global communication.

![image](https://user-images.githubusercontent.com/347290/110020136-39000100-7cf7-11eb-9fb8-84450ae68ea5.png)

## Other Ideas

If you have other ideas on how to visually iconify a decentralized, peer to peer, censorship resistant communication platform then please feel free to suggest a direction. Also please work with a spirit of cooperation rather than competition.

**Comment by miron** [comment]
Imagine two Wardenclyffe towers playing the Super Mario Brothers theme song like in this video 😮


[![](https://img.youtube.com/vi/B1O2jcfOylU/0.jpg)](https://www.youtube.com/watch?v=B1O2jcfOylU)

**Comment by Z64DD** [comment]
Going to listen to Music for the Masses and think about this Clarion OS logo. 
PS great project. 
![9AD8C06E-6B8F-49E1-9911-11954BE2AF18](https://user-images.githubusercontent.com/80068378/110029239-0cdd8380-7ce9-11eb-888c-b3d972537ba7.jpeg)

**Comment by UrsaPolarisRecords** [comment]
Sounds like something I can actually help with! I’ll get on it when I get off work

**Comment by Mim800** [comment]
Here a proposal

**Comment by Mim800** [comment]
Here a proposal

**Reply by jbuck84** [reply]
I’m digging the one w the black background. Could you maybe take that other image you have and have it build off the ‘o’ in clarion?

**Reply by miron** [reply]
Second one is perfect, but Thomas Edison wouldn’t approve the alternating current symbol 😆

**Reply by miron** [reply]
I would maybe change the lightning bolts a little, people are very allergic to the slightest similarity to runic symbols nowadays…

**Reply by codynem** [reply]
<img width="508" alt="Screen Shot 2021-03-04 at 4 58 57 PM" src="https://user-images.githubusercontent.com/25758793/110064942-564ec280-7d2b-11eb-9af3-1e2a85cc1401.png">
<img width="635" alt="Screen Shot 2021-03-04 at 4 36 58 PM" src="https://user-images.githubusercontent.com/25758793/110064963-61a1ee00-7d2b-11eb-94e1-c828ec80a11a.png">

**Reply by codynem** [reply]
I'm not actually a graphic designer, those are just stock icons lol, just a couple riffs for ya;)

**Comment by BenHigh5** [comment]
Link to repo with license - https://github.com/tipitio/LogoSubmission
![image](https://user-images.githubusercontent.com/9750999/110047729-41f6cf80-7d03-11eb-9750-2275ed33fe8b.png)

**Reply by jbuck84** [reply]
Looks pharmaceutical. But I like the direction!

**Reply by bytemaster** [reply]
It took me a while to see the trumpet.

**Reply by BenHigh5** [reply]
> It took me a while to see the trumpet.

I'm glad you saw it. I was going for an abstract idea where it was a trumpet, but also the ground, sun and sky (it can also be seen as representing a clear day), but it just didn't make sense to have a yellow mist in the air, and there are white trumpets as well as gold.

**Reply by BenHigh5** [reply]
Here's a variation on the idea with simple coloring and an Arial like font :
![image](https://user-images.githubusercontent.com/9750999/110154172-cc3d4300-7d98-11eb-8686-45c3668b4952.png)

**Comment by RodrigoMazza** [comment]
![clarion](https://user-images.githubusercontent.com/80078020/110051389-88b0ed80-7d34-11eb-85d9-7f714d98cee8.jpg)

**Reply by bytemaster** [reply]
I like the direction.

**Reply by RodrigoMazza** [reply]
![clarion2](https://user-images.githubusercontent.com/80078020/110066909-4ba72400-7d51-11eb-9048-72c2661ce177.jpg)

**Reply by ChuckMacDonald** [reply]
really nice!

**Reply by bytemaster** [reply]
I really like this concept.

**Reply by RodrigoMazza** [reply]
Thanks! Here another way:
![clarion3](https://user-images.githubusercontent.com/80078020/110130128-7623cc00-7da7-11eb-81d6-81bfcde16a8d.jpg)

**Reply by Nudimar** [reply]
![](https://user-images.githubusercontent.com/80078020/110066909-4ba72400-7d51-11eb-9048-72c2661ce177.jpg)

I've been thinking about the logo in the last days and honestly I think you came up with the best concept.

**Reply by Nudimar** [reply]
Why do I think this is a good hypothesis for the logo?
- number three (Fibonacci)
- allusion to waves
- frequency
- energy
- triangle
- Clarion C

I like your concept. I have several ideas that I would like to bring together, but it is difficult not to overburden with complexity(the best part is the non part) . Well I think that this is where the line of separation of a professional and an amateur begins. but I will keep thinking about more logos. Proud to participate

**Reply by RodrigoMazza** [reply]
Thank Nudimar you for your comments. I am working on a new version of the same concept.

**Reply by jenkinrocket** [reply]
> I really like this concept.

This one has my vote as well. If feels like if symbolizes something dynamic and new, yet also sticks to the themes of sound, communication, and 'making waves'.

**Comment by jordanmasters** [comment]
Going for that broadcasting-to-a-small-group / listening vibe. The third broadcasting-arch wraps around (in the big C) and creates a local community/network. Lot of simple variations.

<img width="395" alt="clarion_logo11" src="https://user-images.githubusercontent.com/4753293/110061576-a7a38580-7d1c-11eb-95b7-ecc1f0c261ef.png">


<img width="126" alt="clarion_logo6" src="https://user-images.githubusercontent.com/4753293/110059203-9c4e5b00-7d18-11eb-84cd-193313619eec.png">

<img width="201" alt="clarion_logo10_pair" src="https://user-images.githubusercontent.com/4753293/110060884-7aa2a300-7d1b-11eb-86b4-12db16dc3b70.png">


<img width="258" alt="clarion_logo9_triangle" src="https://user-images.githubusercontent.com/4753293/110060697-2b5c7280-7d1b-11eb-8cc1-461529bfd56f.png">

**Comment by mingxinyiriyi** [comment]
**The horn establishes a connection between people and spreads the voice of the world**


![clarion](https://user-images.githubusercontent.com/15307525/110052964-7023f000-7d93-11eb-8d4a-f56e4c8ce591.png)

**Reply by jbuck84** [reply]
I was trying to come up w a conch shell design, this makes me think of it again for some reason

**Comment by Nudimar** [comment]
![Clarion](https://user-images.githubusercontent.com/35969153/110056193-413b5900-7d56-11eb-9576-cf6b6dd5109a.png)

**Reply by bytemaster** [reply]
Too complex, imagine this in your doc.

**Reply by ChuckMacDonald** [reply]
![ClarionOS Logo1-Twitter](https://user-images.githubusercontent.com/62088586/110060998-2c9e9700-7d3d-11eb-9faf-504caf20726b.png)

A mashup of two open source icons

**Reply by ChuckMacDonald** [reply]
https://www.google.com/search?hl=en-CA&tbs=simg:CAES8wEJBHAmRqv6oPga5wELELCMpwgaOgo4CAQSE4sKmjzYC5Qw-hrUH9g6SroN8BIaG9DCmLiRYmxO_1kU6Mgrvrs_1FtXc5xxv-mUz1iyAFMAQMCxCOrv4IGgoKCAgBEgTTjMlhDAsQne3BCRqHAQobCgh2ZXJ0aWNhbNqliPYDCwoJL2EvNGhoM3AwChYKA2RvdNqliPYDCwoJL20vMDI3Y3RnChsKCGxhbmd1YWdl2qWI9gMLCgkvai8yc2hfeTQKGQoGbG92ZWx52qWI9gMLCgkvYS9kemtoZl8KGAoGY2lyY2xl2qWI9gMKCggvbS8wMXZrbAw&sxsrf=ALeKk00-ycHH_lEVOb4okYu-rx8P-5YmZw:1614912484502&q=vertical&tbm=isch&sa=X&ved=2ahUKEwiLnO3nkZjvAhUKh-AKHdDYABYQjJkEegQIBhAB&biw=1745&bih=855&dpr=1.1

A preliminary image search.. It looks really familiar to me, but I don't know if it exists.

**Reply by bytemaster** [reply]
Cool icon, would probably look good in a UI element. Not sure about a logo.  I suspect there may be something to this direction.

**Comment by maximusmaximus** [comment]
Here's my contributions to the project. I could prob make these all day. Let me know if you would like to see any revisions or changes. EOS:[max.eosn](https://bloks.io/account/max.eosn)
![image](https://user-images.githubusercontent.com/15719015/110071193-76cd4b80-7d30-11eb-9c91-a5b302827eb6.png)

**Reply by UrsaPolarisRecords** [reply]
![Clarion OS2](https://user-images.githubusercontent.com/36178664/110075418-a16ed280-7d37-11eb-84c9-5387e3241afb.png)

My quick sketch of what I was thinking

**Reply by miron** [reply]
> Here's my contributions to the project. I could prob make these all day. Let me know if you would like to see any revisions or changes. EOS:[max.eosn](https://bloks.io/account/max.eosn)
> ![image](https://user-images.githubusercontent.com/15719015/110071193-76cd4b80-7d30-11eb-9c91-a5b302827eb6.png)

Nr. 3 is definitely a no due the resemblance to a certain emoji, I already can see the memes :)

**Reply by bytemaster** [reply]
Top left seems most "descriptive", the bolts seem "sharp" and "dangerous" and "unfriendly".

**Reply by maximusmaximus** [reply]
Definitely got inspired from some of these other contributions to do another round:
![image](https://user-images.githubusercontent.com/15719015/110234079-758d5180-7edd-11eb-9b46-8f020a561303.png)

**Comment by cevizlibaklava1** [comment]
Inclusive...[
![CLARION](https://user-images.githubusercontent.com/80094197/110087045-314a6600-7da4-11eb-967e-026332dcd880.png)
](url)

**Reply by bytemaster** [reply]
I would try to avoid including the word clarion in the logo/icon.

**Comment by CryptoCutter** [comment]
Just a rough draft
![Shape 1_20210305_023629](https://user-images.githubusercontent.com/35167878/110089661-fcb2bc00-7d5b-11eb-9b37-2cd41dfb8dc5.png)

**Reply by bytemaster** [reply]
Interesting concept.  If feels empty in the middle, but it might just be the scale is so huge :)

**Reply by maximusmaximus** [reply]
Reminds me of elephant tusks for some reason

**Reply by CryptoCutter** [reply]
It definitely would get something in the middle, I was thinking a face maybe, something that will look simple and definable as a token symbol, unfortunately I'm out of town for a couple days and don't have my computer with me to work on it, thought I would throw it out for a brainstorm idea for others.

**Reply by CryptoCutter** [reply]
> Reminds me of elephant tusks for some reason

It does, a basic elephant face in the middle might work...

**Comment by wiedzmak** [comment]


**Reply by UrsaPolarisRecords** [reply]
Here's an alteration to the design. The intention is to use a subtle trumpet shape to communicate the single node broadcasting through other nodes, and then on to other nodes. I personally like this one better than the one I submitted earlier.
![Clarion OS3](https://user-images.githubusercontent.com/36178664/110098106-aa21d180-7d54-11eb-94de-6130d7ad873f.png)

**Reply by UrsaPolarisRecords** [reply]
same design, different colors:
![Clarion OS3-01](https://user-images.githubusercontent.com/36178664/110108438-0a1e7500-7d61-11eb-8593-9fd83a1a07a6.jpg)

**Reply by UrsaPolarisRecords** [reply]
One more in all brass color so that it doesn't look as much like the symbol for Islam, which is fine but is not related to this project.
![Clarion OS3-01](https://user-images.githubusercontent.com/36178664/110109988-07248400-7d63-11eb-8574-baa72b958fe1.jpg)

**Reply by bytemaster** [reply]
I like this direction.  Something seems off about the left side where arches get "thicker".

**Reply by Mabims** [reply]
> My take, clarion as the network:
> 
> ![image](https://user-images.githubusercontent.com/80074132/110091570-e24b0380-7d98-11eb-9b4a-dc83684df86e.png)
> ![image](https://user-images.githubusercontent.com/80074132/110091585-e5de8a80-7d98-11eb-8d96-590b2c1768e2.png)

This is cool

**Comment by hyp3rn01d** [comment]
![Clarion](https://user-images.githubusercontent.com/46609818/110104013-eaaa3b00-7da6-11eb-8b80-744bbef385af.png)

![Clarion_OS](https://user-images.githubusercontent.com/46609818/110104069-fac21a80-7da6-11eb-857a-fa7929a7ae70.png)

**Reply by bytemaster** [reply]
Too big for an icon.  I appreciate the idea though.

**Reply by hyp3rn01d** [reply]
![Clarion_C_2021_03_05](https://user-images.githubusercontent.com/46609818/110133476-a29e0f00-7dcc-11eb-9367-b133ea9b9476.png)

**Comment by cevizlibaklava1** [comment]
![CLARION2](https://user-images.githubusercontent.com/80094197/110111331-34a01a80-7dc1-11eb-92fd-460cae8cfe86.png)
indifferent to the source and inclusive of all the voices

**Comment by Echoes-1** [comment]
Hi, this is my first proposal. The final part of a clarion, in negative space, forms a C letter. You can also use the "petals" as a pattern.

<img width="600" alt="clarionos" src="https://user-images.githubusercontent.com/80106056/110118302-737ea200-7dba-11eb-83c2-c737d26ea022.png">

**Reply by BenHigh5** [reply]
I like the simplicity of this one, it's really clean and nice.

**Reply by Echoes-1** [reply]
![clarionos_sphere](https://user-images.githubusercontent.com/80106056/110200453-76cb6980-7e5e-11eb-98ca-468141751049.png)
A variation on theme: working on the left one, it becomes a sphere (globe -> global; in some way the upper part of the Wardenclyffe tower also). So: letter C, hidden clarion, globe.

**Comment by Mim800** [comment]


**Reply by Mim800** [reply]
It's the modern Agorà on which different elements converge. Elements could be seen as bells, flowers, stylus nibs, trumpets, cuneiform characters, chestahedrons... 5 star represents human being... Purple is the color of edge of chaos, emergent steady state between order and chaos (see Prigogine experiments)

**Comment by Dexaran** [comment]
cc @Sparke2

**Comment by jiggabola** [comment]
one of my ideas 
![Clarion022](https://user-images.githubusercontent.com/41899470/110151177-d5560080-7de8-11eb-93c3-1a3a0c1406e2.jpg)
![Clarion011](https://user-images.githubusercontent.com/41899470/110151183-d71fc400-7de8-11eb-8e76-5de6ef0e73ba.jpg)

**Comment by BRCreative** [comment]
![Clarion_04](https://user-images.githubusercontent.com/32405250/110157963-a6666d00-7d9d-11eb-9e1a-c58037ebf4e8.jpg)
![Clarion_Select](https://user-images.githubusercontent.com/32405250/110157968-a7979a00-7d9d-11eb-8159-273bf9e06c87.jpg)
Hi Group,

Here's my initial concept for the logo. Clean and simple.
A few choices there. Hard to decide which one I prefer.

**Comment by RodrigoMazza** [comment]
![clarion4](https://user-images.githubusercontent.com/80078020/110163820-56a09980-7dcf-11eb-9a35-9d78e438d412.jpg)

**Comment by Sparke2** [comment]
![сlarion_logo](https://user-images.githubusercontent.com/29039716/110177857-07825500-7e1f-11eb-974a-1f00c303d882.png)

A set of logos reflecting the decentralized nature of the project.
All images are CC0 licensed: https://github.com/Sparke2/ClarionOS_logo

**Reply by BenHigh5** [reply]
I like that you brought decentralization into it.

**Comment by ChuckMacDonald** [comment]
That's a really nice looking horn. Love it.


<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by gaijinjoe** [comment]
our 3 takes
![128880955011a03d22664bf047d7c98](https://user-images.githubusercontent.com/23469584/110200929-62f22880-7e9b-11eb-93fa-104f1ecad739.png)
![2dca3c6df64836d72f91e22e37f5299](https://user-images.githubusercontent.com/23469584/110200933-67b6dc80-7e9b-11eb-940c-1ddcd4c93c7a.png)
![bb03a9e33fc2920ddc4de1a97eccdf6](https://user-images.githubusercontent.com/23469584/110200935-6b4a6380-7e9b-11eb-87ee-f069b53adcd3.png)

**Comment by budiibud** [comment]
new idea
![CLARION#02](https://user-images.githubusercontent.com/80144191/110204520-1102d000-7ea6-11eb-9ebe-8ab1a92edb2e.jpg)

**Reply by budiibud** [reply]
Clean & dynamic!

**Reply by budiibud** [reply]
option as a wordmark logo
![CLARION#02b](https://user-images.githubusercontent.com/80144191/110204803-a6eb2a80-7ea7-11eb-9b51-630c64a37e7d.jpg)

**Comment by cabondon** [comment]
My take on that.
![ClarionOS](https://user-images.githubusercontent.com/80120160/110204613-dc722280-7e6b-11eb-8e26-0636bd29aa6c.png)

![ClarionOS bnw](https://user-images.githubusercontent.com/80120160/110204722-7d60dd80-7e6c-11eb-9e5e-c02f3d84f713.png)

**Comment by RodrigoMazza** [comment]
new version:
![clarion5](https://user-images.githubusercontent.com/80078020/110207375-e000ac00-7e61-11eb-8365-f32049dfc6ab.jpg)

**Reply by bytemaster** [reply]
I like this direction too.

**Comment by Sparke2** [comment]
A small update

![c_13](https://user-images.githubusercontent.com/29039716/110211132-9c805f00-7eae-11eb-8408-3373d0fa3fb5.png)
![c_14](https://user-images.githubusercontent.com/29039716/110211134-9d18f580-7eae-11eb-9ce6-fd24fcb46c68.png)

**Comment by th3nyst1c** [comment]
![ClaironOS](https://user-images.githubusercontent.com/80164766/110211950-553ba380-7e99-11eb-911d-0b793ab3e8fb.jpg)

**Comment by th3nyst1c** [comment]
![CLARION2](https://user-images.githubusercontent.com/80164766/110213635-20cbe580-7ea1-11eb-930c-1b4f08d8c5bf.png)

**Comment by cabondon** [comment]
More relatable take.
![ClarionOs](https://user-images.githubusercontent.com/80120160/110215439-c2533700-7ea1-11eb-8bb4-cd6d312f2a3a.png)

**Comment by CryptoCutter** [comment]
![Clarion_Plus](https://user-images.githubusercontent.com/35167878/110218443-6bd50100-7e7f-11eb-94df-7883a311859c.png)

I'm going for a clickable logo that is still definable in a thumbnail

**Comment by RodrigoMazza** [comment]
![clarion6](https://user-images.githubusercontent.com/80078020/110222642-73fa6400-7eb2-11eb-8026-8b96ba3785f0.jpg)

**Comment by budiibud** [comment]
new logo idea
![CLARION#04](https://user-images.githubusercontent.com/80144191/110225441-2e708200-7f18-11eb-8476-3308fa167921.jpg)
Trumpet+C shape(Clarion)+the sound wave+the world
simple,clean and dynamic logo

**Reply by CryptoCutter** [reply]
That's really good, love that color gradient too!

**Reply by budiibud** [reply]
Thank you, brother!

**Reply by chizzy123** [reply]
wow great job!

**Comment by RodrigoMazza** [comment]
![clarion7](https://user-images.githubusercontent.com/80078020/110227367-84bcd100-7ed6-11eb-939a-55e7a2647da6.jpg)

**Reply by adrian-cap** [reply]
Yep, this is the best version. I really like it.

**Reply by bytemaster** [reply]
I think it loses something on anything other than white background. The clarion almost disappeared at small size.

**Reply by RodrigoMazza** [reply]
It is true. Here is a version with a more readable typeface:
![clarion7b](https://user-images.githubusercontent.com/80078020/110244641-6d193300-7f3e-11eb-856c-1a2f02f7e194.jpg)

**Comment by budiibud** [comment]
clean, simple, iconic and modern logo
![CLARION#06](https://user-images.githubusercontent.com/80144191/110236800-6f918200-7f6a-11eb-9072-0ac0b2f19a88.jpg)

**Reply by maximusmaximus** [reply]
This is my favorite so far. Great user of deconstruction. Thinking solid colors without the gradient would be better. When it gets small it won't get muddy.

**Reply by budiibud** [reply]
Thanks, maximusmaximus! Yes, I think your suggestion is very good👍 it can be color options for the logo.

**Comment by budiibud** [comment]
another option,similar concept like before
There's a hidden trumpet shape in the logo
![CLARION#07](https://user-images.githubusercontent.com/80144191/110236901-56d59c00-7f6b-11eb-99df-fef002e811ea.jpg)

**Comment by RodrigoMazza** [comment]
![clarion8](https://user-images.githubusercontent.com/80078020/110242862-ab125900-7f36-11eb-9b75-21e2ef5d1bd6.jpg)

**Reply by CryptoCutter** [reply]
I like this interconnected logo style

**Comment by Arcan-Hell** [comment]
![clarion 1](https://user-images.githubusercontent.com/80136128/110258570-a15e1500-7f79-11eb-8af2-47536d74fb66.jpg)

**Comment by alphacrypto** [comment]
![clarion-austin-infinite-wireless-network](https://user-images.githubusercontent.com/26175152/110262113-7b2f7980-7f67-11eb-8ac9-54cb700f5590.png)

i think you need the logo city-based to encourage city effects

plus just 'clarion' is fine and easier / more stylish / more friendly to hot smart good girls

**Comment by CryptoCutter** [comment]
Color balance is off, but I figure I would put it out before turning in for the night.![image](https://user-images.githubusercontent.com/35167878/110262319-efbee400-7f78-11eb-8f9c-0df82f1034e5.png)

**Comment by maximusmaximus** [comment]
![110234079-758d5180-7edd-11eb-9b46-8f020a561303](https://user-images.githubusercontent.com/15719015/110268444-7cb46e00-7f76-11eb-97c4-0981a2c7f4c6.png)

**Comment by cryptart-1** [comment]
![image](https://user-images.githubusercontent.com/80230960/110269584-12112b80-7f92-11eb-88a8-436afdf24218.jpeg)

![image](https://user-images.githubusercontent.com/80230960/110269598-19383980-7f92-11eb-984a-355762064490.jpeg)

Letter "C" font that creates trumpets at both ends while keeping simplicity. 

Lines represent sheet music. Metaphor for Clarion being heard by the world. 
Also gives it the look of a valuable antique guitar. Another metaphor for spreading Clarions voice to the world.

**Reply by ConsciousCreator22** [reply]
Love this design and represents the meanings. Good work !

**Comment by kommienezuspadtOS** [comment]
Allocated some time to work on a potential logo for Clarion tonight. Based off of the Telsa coil reference image above. I didn't add any text, but could easily be included if necessary. 
![ClarionLogo002](https://user-images.githubusercontent.com/80239439/110278295-9cf72380-7f9c-11eb-965b-aa8a6aef621a.jpg)

**Comment by jiggabola** [comment]
Works with different colors as background
![Clarion OS](https://user-images.githubusercontent.com/41899470/110340274-c01fe280-8031-11eb-9cc9-f3d7fb1f7fc6.jpg)

**Comment by algorithmic-cage** [comment]
<a href="https://ibb.co/TWSGGvY"><img src="https://i.ibb.co/VwZGGjJ/First-Try01.png" alt="First-Try01" border="0"></a>

**Reply by algorithmic-cage** [reply]
Use CTRL+Mousewheel to see the use for small scale icons.
I hope that people see the clarion in "fish avatar" and in the owl / trilobite.

**Comment by kommienezuspadtOS** [comment]
I worked on this variation tonight, off the above mentioned source, admittedly. I was talking with friends in my circle and someone mentioned the ClarionOS project as a "dark horse", especially in regards to the possibly of an EOS fork. The idea stayed with me and nagged until I did this illustration just to get it off my mind.

![ClarionDarkHorse001](https://user-images.githubusercontent.com/80239439/110426229-58868900-806b-11eb-8c3c-595ba16d7494.jpg)

**Reply by algorithmic-cage** [reply]
wow

**Comment by budiibud** [comment]
combination of The trumpet+Tesla lightning+The globe(world)
![CLARION#08](https://user-images.githubusercontent.com/80144191/110429588-bf448a80-80dd-11eb-8332-7082c32365a9.jpg)

**Reply by BenHigh5** [reply]
I like the trumpet/lightning combo, that is good.

**Reply by maximusmaximus** [reply]
The trumpet and bolt as a mirror then reflected to produce a figure 8, might be visually interesting

**Reply by CryptoCutter** [reply]
I really like this one, but it might lose some definition in thumbnail form.

**Reply by budiibud** [reply]
Thanks for your comments and suggestions, guys!🙏

**Reply by eosdev-x** [reply]
I think this one is my favorite so far! Very clean!

**Reply by adrian-cap** [reply]
It looks like a knights side profile.

**Comment by kommienezuspadtOS** [comment]
Based on the Horn of Gondor from LOTR’s. Don’t judge.😂

![IMG_3466](https://user-images.githubusercontent.com/80239439/110522641-b8fee000-80d6-11eb-907e-7451418b5cdf.jpeg)

**Reply by kommienezuspadtOS** [reply]
Black and white version as well.

![IMG_3466](https://user-images.githubusercontent.com/80239439/110522946-1bf07700-80d7-11eb-81e5-d05c643293cf.jpeg)

**Comment by budiibud** [comment]
Simple and iconic logo. The logo is combination of the megaphone+lightening+textbubble symbol(means: message to declare something).
![CLARION#09](https://user-images.githubusercontent.com/80144191/110778667-3a03d600-8295-11eb-8750-bf44ce86ec6d.jpg)

**Reply by maximusmaximus** [reply]
Like this, it's my favorite one on here. The bolt reminds me on LN which many people in these crypto communities might make assumptions about this protocol running in LN

**Reply by cmjteale** [reply]
Clear, iconic, and boldly representing the message. As an icon on a users phone it probably wont outdate in looks for a while.
Looking forward to see how Clarion goes.

**Comment by bludyroar** [comment]
![clarion_logo_final_2a](https://user-images.githubusercontent.com/80460851/110793713-3c563800-8275-11eb-831a-ce591ecadb19.jpg)

**Comment by bludyroar** [comment, marked as answer]
![clarion_logo_8a_3colors](https://user-images.githubusercontent.com/80460851/110801460-90fdb100-827d-11eb-9291-ddac36d23564.jpg)

**Reply by BenHigh5** [reply]
very cool concept.

**Reply by bytemaster** [reply]
Please contact me on twitter or telegram (bytemaster7) if you would like to accept a gift for your generous contribution.

**Reply by bludyroar** [reply]
Wow!! Hey Dan. Me and my brother made that logo together. We are really excited, and had a little celebration when we saw your message! 
We are both big eos supporters since we saw an interview of you, before eos was launched. Your vision is the reason we are heavily involved in crypto to change to world for the better. Glad to see you back in the community.

The logo still need some adjustments. Some lines need some tweaking, to make it perfect. 

Roar & Giordano

**Comment by RodrigoMazza** [comment]
![clarion9](https://user-images.githubusercontent.com/80078020/110829261-ad4a1f80-8276-11eb-8c80-d64d5384ddaa.jpg)

**Reply by maximusmaximus** [reply]
Very cool design! inner element similar to the acrobat logo.

**Reply by thomas-hallgren** [reply]
the ribbon in the center reminds me of the Adobe logo.

**Comment by CryptoCutter** [comment]
![Clarion_Bloom](https://user-images.githubusercontent.com/35167878/110889989-cd98cf00-82b4-11eb-899d-fa86a3f223f3.png)

**Comment by CryptoCutter** [comment]
No center line ![Clarion_Bloom_Solid](https://user-images.githubusercontent.com/35167878/110891086-dee2db00-82b6-11eb-9c89-c7c0bfa88997.png)

**Comment by RodrigoMazza** [comment]
![clarion11](https://user-images.githubusercontent.com/80078020/111029762-295e7780-83dd-11eb-9d35-0d55e63399e9.jpg)

**Comment by ladybond** [comment]
![image3](https://user-images.githubusercontent.com/79706323/111038933-7e7da600-843c-11eb-95ca-fa1a6e447699.jpeg)
![image2 (1)](https://user-images.githubusercontent.com/79706323/111038934-7f163c80-843c-11eb-988a-c9e3a055a4ce.jpeg)
![image1](https://user-images.githubusercontent.com/79706323/111038935-7f163c80-843c-11eb-9c18-47ce1b3f090d.jpeg)
![image0 (2)](https://user-images.githubusercontent.com/79706323/111038937-7faed300-843c-11eb-92f8-37633c6a0ba6.jpeg)
@bytemaster

**Reply by RodrigoMazza** [reply]
![clarion12](https://user-images.githubusercontent.com/80078020/111041636-ebcb1000-8417-11eb-88b9-bfd96e48363d.jpg)

**Reply by BenHigh5** [reply]
I love the hidden trumpets/bugles.

**Comment by DaCryptonaut** [comment]
![clarion OS](https://user-images.githubusercontent.com/46240701/111058915-978b5480-8446-11eb-922b-bb88ee36d2c6.png)
![clarion OS4](https://user-images.githubusercontent.com/46240701/111058916-98bc8180-8446-11eb-898c-74992443063d.png)
![clarion OS3](https://user-images.githubusercontent.com/46240701/111058917-99edae80-8446-11eb-90a5-030cc02abad8.png)
![clarion OS2](https://user-images.githubusercontent.com/46240701/111058918-99edae80-8446-11eb-996a-6dbe19379031.png)

with love, from blockspace

**Comment by ladybond** [comment]
![image3 (1)](https://user-images.githubusercontent.com/79706323/111072397-847e9000-84eb-11eb-8a7a-daa49af6085e.jpeg)
![image2 (2)](https://user-images.githubusercontent.com/79706323/111072398-85afbd00-84eb-11eb-81c3-e9906fcb68ea.jpeg)
![image1 (1)](https://user-images.githubusercontent.com/79706323/111072399-85afbd00-84eb-11eb-946b-346fe56f97f3.jpeg)
![image0 (3)](https://user-images.githubusercontent.com/79706323/111072400-85afbd00-84eb-11eb-9001-dbb3d7bf728f.jpeg)
@bytemaster

**Comment by olw48** [comment]
![clarion4](https://user-images.githubusercontent.com/80676466/111130559-16a59780-85c3-11eb-965e-ec2fd1660e74.png)

**Comment by Arcan-Hell** [comment]
Another proposal
![CLARION3](https://user-images.githubusercontent.com/80136128/111252098-cd673f00-85e6-11eb-82e9-c0a37421cbdc.jpg)
@bytemaster

**Comment by kommienezuspadtOS** [comment]
![ClarionLogo005](https://user-images.githubusercontent.com/80239439/111392350-626c4580-8684-11eb-814a-ad50ce441fec.png)

I actually stumbled upon this logo while doing a warm up exercise for another project. Dropped the microphone in and it just worked so much better as a potential ClarionOS logo.

**Comment by DAE2021** [comment]
Hello Daniel Larimer,
I sincerely would like to participate in your new project. I would like to get in touch with you. Is there any way? - from South Korea
